src/ui/core/zcl_abapgit_gui.clas.abap

```diff
@@ -306,9 +307,8 @@ CLASS ZCL_ABAPGIT_GUI IMPLEMENTATION.
       CATCH zcx_abapgit_cancel ##NO_HANDLER.
         " Do nothing = gc_event_state-no_more_act
       CATCH zcx_abapgit_exception INTO lx_exception.
-        IF mi_error_handler IS BOUND.
-          mi_error_handler->handle_error( lx_exception ).
-        ENDIF.
```

Comment:
I'd vote against removing this part (and the local class in the factory). The point was to remove explicit ROLLBACK from GUI - it can potentially block reusing the GUI in other code